test/cpp/ext/otel/otel_plugin_test.cc

```diff
@@ -775,19 +775,15 @@ class CustomPluginOption
 };
 
 TEST_F(OpenTelemetryPluginOptionEnd2EndTest, Basic) {
-  std::vector<
-      std::unique_ptr<grpc::internal::InternalOpenTelemetryPluginOption>>
-      plugin_option_list;
-  plugin_option_list.emplace_back(std::make_unique<CustomPluginOption>(
-      /*enabled_on_client*/ true, /*enabled_on_server*/ true,
-      std::make_pair("key", "value")));
   Init(
       std::move(Options()
                     .set_metric_names({grpc::OpenTelemetryPluginBuilder::
                                            kClientAttemptDurationInstrumentName,
                                        grpc::OpenTelemetryPluginBuilder::
                                            kServerCallDurationInstrumentName})
-                    .set_plugin_options(std::move(plugin_option_list))));
+                    .add_plugin_option(std::make_unique<CustomPluginOption>(
```

Comment:
I like the change to use `add_plugin_option()` instead of having to construct a vector!